Mathlib/Algebra/Order/Group/Cyclic.lean

```diff
@@ -53,6 +54,15 @@ lemma genLTOne_lt_one : H.genLTOne < 1 :=
 lemma genLTOne_zpowers_eq_top : Subgroup.zpowers H.genLTOne = H :=
   H.exists_generator_lt_one.choose_spec.2
 
+lemma genLTOne_unique (g : G) : g < 1 ∧ Subgroup.zpowers g = H → g = H.genLTOne := by
```

Comment:
I think this should be formulated as follows:
```lean
lemma genLTOne_unique (g : G) (hg : g < 1) (hH : Subgroup.zpowers g = H) : g = H.genLTOne := by
```

Also, could you please add a lemma of the following form?
```lean
lemma genLTOne_unique (g1 g2 : G) (hg1 : g1 < 1) (hg2 : g2 < 1) (h : Subgroup.zpowers g1 = Subgroup.zpowers g2) : g1 = g2 := by
```

(please also make the analogous changes in the `IsCyclic` section.)